psi4/src/psi4/libscf_solver/sad.cc

```diff
@@ -488,21 +488,33 @@ void SADGuess::get_uhf_atomic_density(std::shared_ptr<BasisSet> bas, std::shared
     // Setup JK
     std::unique_ptr<JK> jk;
 
+    std::string jk_type(options_.get_str("SAD_SCF_TYPE"));
+
+    // Handle default cases for compatibility
+    if ((jk_type == "PK") || (jk_type == "OUT_OF_CORE")) {
+        jk_type = "DIRECT";
```

Comment:
This new strategy looks good to me. It hits the user a little bit, but overall makes sense I think as it gives them effectively what they asked for.